### DISCUSSION TITLE
What is the recommended upgrade method?

### DISCUSSION THREAD

**Discussion by viharm** [firefly-iii/firefly-iii#5794] [Firefly III] [answered]
I'm using _Firefly III_ as a _Docker_ image.

I'm currently at 5.6.10, & want to upgrade to the latest version (currently 5.6.14).

Is there a recommended way or should I simply rerun the container by pulling the latest image?

PS: Backup of DB and volume will obviously be done

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/firefly-iii/installation/upgrade/

**Reply by viharm** [reply]
It was right there, well-documented. Thanks.
